auth4genai/snippets/get-started/langchain-next-js/call-others-api.mdx

```diff
@@ -386,9 +386,9 @@ To implement, install the Auth0 AI Components for Next.js SDK to get the require
 npx @auth0/ai-components add TokenVault
 ```
 
-Add a new file, `src/components/auth0-ai/TokenVault/TokenVaultInterruptHandler.tsx`, with the following code:
```

Comment:
reflecting the fact that we moved this file to root of `src/components` (missed in previous wave of edits)